libs/cardano-protocol-tpraos/cardano-protocol-tpraos.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 3.0
 name:          cardano-protocol-tpraos
-version:       1.0.1.0
+version:       1.0.2.0
```

Comment:
`1.0.2.0` is already on CHaP
```suggestion
version:       1.0.3.0
```

Reply:
good catch, thank you!

Reply:
I will fix this in the PR to master (#3410) as well